src/sentry/snuba/subscriptions.py

```diff
@@ -120,7 +127,7 @@ def update_snuba_query(
         )
 
 
-def bulk_create_snuba_subscriptions(projects, subscription_type, snuba_query):
+def bulk_create_snuba_subscriptions(projects, subscription_type, alert_rule):
```

Comment:
We shouldn't be passing the alert rule down to this level, this system doesn't know alert rules exist.